lib/tasks/learn_to_rank.rake

```diff
@@ -56,9 +56,11 @@ namespace :learn_to_rank do
       bucket_name = ENV["AWS_S3_RELEVANCY_BUCKET_NAME"]
       raise "Missing required AWS_S3_RELEVANCY_BUCKET_NAME" if bucket_name.blank?
 
-      raise "Please specify the model filename" if args.model_filename.blank?
+      model_files = Aws::S3::Bucket.new(bucket_name).objects.map(&:key)
 
-      model_filename = args.model_filename
+      model_filename = args.model_filename || model_files.max
```

Comment:
I suspect that given the option between `9.zip` and `10.zip` then it'll choose `9.zip` as the `max`.  We might need to sort by some function instead so that we can recognise the file names as being numbers and not strings?

Reply:
Good point @sihugh!

Reply:
```
files = ["1.zip", "9.zip", "3.zip", "10.zip"]
=> ["1.zip", "9.zip", "3.zip", "10.zip"]

files.max_by { |f| f.to_i }
=> "10.zip"
```